tests/test_modeling_utils.py

```diff
@@ -724,7 +724,6 @@ def test_from_pretrained_hub_pass_model(self):
 
         # pass unet into DiffusionPipeline
         unet = UNet2DModel.from_pretrained(model_path)
-        ddpm_from_hub_custom_model = DDPMPipeline.from_pretrained(model_path, unet=unet)
```

Comment:
Ok for me! I did this actually someone on purpose to make sure both `DDPMPipeline` and `DiffusionPipeline` work with it, but `DiffusionPipeline` is def the more important one